src/FantomasVsPackage.cs

```diff
@@ -11,7 +11,7 @@
 namespace FantomasVs
 {
 
-    // DO NOT REMOVE THIS MAGICAL INCANTATION NO MATTER HOW MUCH VS WARNS YOU OF DEPRECATION    
+    // DO NOT REMOVE THIS MAGICAL INCANTATION NO MATTER HOW MUCH VS WARNS YOU OF DEPRECATION
```

Comment:
I forgot to disable the whitespace remover extension... Let me know if you want me to revert the changes here.

Reply:
All good 👍